src/ontology/edits/oeo-physical.omn

```diff
@@ -2570,8 +2570,10 @@ Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000332>
 Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000333>
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Power is the rate of doing work, the amount of energy transferred per unit time.
-Source: https://en.wikipedia.org/wiki/Power_(physics)"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Power is the quantity that represents the amount of energy transformed per time unit"@en,
```

Comment:
I missed the discussion on Tuesday but why not _Power is the quantity that represents the amount of energy **transferred or** transformed per time unit._

Or is it intended that energy transport is also a kind of transformation? You can state the power not only of a transformer like a generator, but also the power on a power line.

Reply:
We also discussed this in the meeting. Transfer over a powerline was also seen as a transformation. Anyway, I like your suggestion.